business-central-parent/business-central-distribution-wars/src/main/assembly/component-kie-wb-standalone-zip.xml

```diff
@@ -17,18 +17,18 @@
 
 <component xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.3 http://maven.apache.org/xsd/component-1.1.3.xsd">
-  <dependencySets>
+  <!--<dependencySets>
```

Comment:
remove this?

Reply:
yes, that is my intention, but I still kept it there for having it here as a note for follow up PR in another repo.